run_full_test_suite

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
Don't we want to use some virtual env for this?

Reply:
Quick and dirty script for the lazy :) to be improved later